weather_mv/loader_pipeline/bq.py

```diff
@@ -143,10 +146,23 @@ def to_table_schema(columns: t.List[t.Tuple[str, str]]) -> t.List[bigquery.Schem
     fields.append(bigquery.SchemaField(DATA_IMPORT_TIME_COLUMN, 'TIMESTAMP', mode='NULLABLE'))
     fields.append(bigquery.SchemaField(DATA_URI_COLUMN, 'STRING', mode='NULLABLE'))
     fields.append(bigquery.SchemaField(DATA_FIRST_STEP, 'TIMESTAMP', mode='NULLABLE'))
+    fields.append(bigquery.SchemaField(S2_LOCATION, 'GEOGRAPHY', mode='NULLABLE'))
 
     return fields
 
 
+def fetch_s2_location(lat, long):
```

Comment:
As mentioned above, this is not an s2 location, but a "geopoint".

Reply:
Please add type annotations to the function.

Reply:
Type annotations added to said function.